### ISSUE TITLE
fastlane match doesn't create a Provisioning Profile after adding 'Sign in with Apple' capability

### ISSUE THREAD

**Issue by idreamer** [fastlane/fastlane#15353]
### New Issue Checklist

- [/] Updated fastlane to the latest version
- [/] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [/] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [/] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I added 'Sign in with Apple' capability and I revoked the existed certificate and provisioning profile using `match nuke distribution` Then I created a new certificate and a provisioning profile using `match appstore`. But it doesn't create a Provisioning Profile on Apple Developer account. 

##### Command executed

`fastlane match nuke distribution`
`fastlane match appstore`

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.131.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| executable-hooks          | 1.6.0        |
| bundler-unload            | 1.0.2        |
| rubygems-bundler          | 1.4.5        |
| bundler                   | 2.0.1        |
| slack-notifier            | 2.3.2        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| terminal-table            | 1.8.0        |
| CFPropertyList            | 3.0.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.6.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.62.0       |
| unf_ext                   | 0.0.7.5      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20180417 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.9.5        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.2        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| faraday_middleware        | 0.13.1       |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.5        |
| jwt                       | 2.1.0        |
| google-cloud-env          | 1.0.5        |
| google-cloud-core         | 1.3.0        |
| digest-crc                | 0.4.1        |
| google-cloud-storage      | 1.16.0       |
| nanaimo                   | 0.2.6        |
| colored2                  | 3.1.2        |
| claide                    | 1.0.2        |
| atomos                    | 0.1.3        |
| xcodeproj                 | 1.11.0       |
| unicode-display_width     | 1.5.0        |
| plist                     | 3.5.0        |
| tty-screen                | 0.6.5        |
| tty-cursor                | 0.6.1        |
| tty-spinner               | 0.9.0        |
| faraday                   | 0.15.4       |
| fastimage                 | 2.1.5        |
| json                      | 2.2.0        |
| dotenv                    | 2.7.2        |
| httpclient                | 2.8.3        |
| multi_json                | 1.13.1       |
| signet                    | 0.11.0       |
| os                        | 1.0.0        |
| memoist                   | 0.16.0       |
| googleauth                | 0.6.7        |
| mime-types-data           | 3.2018.0812  |
| mime-types                | 3.2.2        |
| retriable                 | 3.1.2        |
| declarative-option        | 0.1.0        |
| declarative               | 0.0.10       |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| google-api-client         | 0.23.9       |
| emoji_regex               | 1.0.1        |
| openssl                   | 2.1.0        |
| date                      | 2.0.0        |
| fileutils                 | 1.0.2        |
| ipaddr                    | 1.2.0        |
| psych                     | 3.0.2        |
</details>

*generated on:* **2019-09-22**
</details>

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

**Comment by janpio** [comment]
Are you reporting that creation of profisioning profiles is broken? Or just in specific circumstances? Is this reproducible?

**Comment by idreamer** [comment]
> Are you reporting that creation of profisioning profiles is broken? Or just in specific circumstances? Is this reproducible?

Hi. Previously I used fastlane match successfully before I updated Xcode 11. Then I added 'Sign in with Apple' capability. Then Xcode shows the provisioning profile doesn't include the capability. So I revoked it and tried to create it again. By the way it doesn't create a new Provisioning Profile to Apple Developer account. If my understanding is correct, match should create a new one to Apple Developer account. 

So I think that it can be reproduced in Xcode 11.
